### PR TITLE
Add `show_app_log` option to tail and display logs during application start

### PR DESCRIPTION
**Pull request by aeijdenberg**
(Shoudn't be merged until <https://github.com/contraband/autopilot/pull/43> lands, and then the `curl` command updated to pull from the correct upstream repo)

Fixes <https://github.com/concourse/cf-resource/issues/28>

Also fixes Docker build, which previously uses `jq` features not present in the Alpine image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@ RUN apk add --no-cache curl jq
 RUN mkdir -p /assets
 WORKDIR /assets
 RUN curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github-rel" | tar -xzf -
-RUN url=$(curl -s "https://api.github.com/repos/contraband/autopilot/releases/latest" \
```
Comment (waterlink) [inline Dockerfile]: Are you going to merge your changes (adding tail logs) to upstream of `autopilot`? Would that be enough to get things going here in this PR?

Comment (aeijdenberg) [reply Dockerfile]: We've sent the changes [upstream](https://github.com/contraband/autopilot/pull/43) and also submitted a number of [other fixes](https://github.com/contraband/autopilot/pull/44) to autopilot, however unfortunately the project seems to be starved of attention by the maintainers.

FWIW we've been using our forked version for all new projects for the past 4 weeks or so and it's been working well for us.

**Comment by drnic** [comment]
Published example docker image worked for me

**Comment by aeijdenberg** [comment]
I've rebased on latest `master`, and switched the URL back to the `contraband` release now that patches landed there.

**Comment by vito** [comment]
thanks!

**Comment by JohannesRudolph** [comment]
This is awesome! Minor: the documentation appears to be missing from README.md, right?

**Pull request merged**